Cargo.toml

```diff
@@ -2,6 +2,9 @@
 name = 'pack_signed'
 harness = false
 
+[lib]
+crate-type = ["cdylib", "rlib"]
```

Comment:
Strange looking change. What is the reason>?

Reply:
fixed